tests/test_selenuim.py

```diff
@@ -0,0 +1,60 @@
+import os
+from threading import Thread
+
+import pytest
+from selenium import webdriver
```

Comment:
`selenium` is not a dependency listed in `requirements.txt`. This prevents the project being setup.

This also blocks this review from passing.

Reply:
updated requirements file